src/components/home-page/body-content.jsx

```diff
@@ -0,0 +1,20 @@
+export const BodyContent = () => {
+    return <div className="home-body-content">
+    <div className="body-content">
+        <div className="body-header">
+            Manage your daily tasks with avaay
+        </div>
+        <div className="body-description">
+            avaay timwe is the easiest way to avoid procastinations and increase your productivity
+        </div>
+
+        <div class="button text primary"> Start now </div>
```

Comment:
use semantic HTML tags, 
```suggestion
        <button class="button text primary"> Start now </button>
```